### PR TITLE
Refs #23919 -- Removed pysqlite Python 2 bits.

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

**Comment by timgraham** [comment]
`grep -rI pysqlite *` still returns results in code and docs.

**Comment by auvipy** [comment]
yes thanks updating

**Comment by auvipy** [comment]
I am not sure about some workaround for pysqlite which seems confusing

**Comment by timgraham** [comment]
In the Python source code, `Modules/_sqlite` is referred to as "pysqlite" so I think some of your removals were overzealous. @claudep, do you have expertise to review this?

**Comment by auvipy** [comment]
yes I was quite confuzed about some removal and wording

**Comment by claudep** [comment]
Historically, I think we always supported either the Python built-in sqlite3 lib, or the external pysqlite module (https://pypi.python.org/pypi/pysqlite).
Reading https://github.com/ghaering/pysqlite/blob/master/setup.py, it looks like pysqlite doesn't support Python 3. So I think this patch is doing the right thing.

File: django/db/backends/sqlite3/base.py [hunk 1]
```diff
@@ -20,14 +18,6 @@
...
 
-try:
-    try:
-        from pysqlite2 import dbapi2 as Database
-    except ImportError:
-        from sqlite3 import dbapi2 as Database
-except ImportError as exc:
-    raise ImproperlyConfigured("Error loading either pysqlite2 or sqlite3 modules (tried in that order): %s" % exc)
-
 # Some of these import sqlite3, so import them after checking if it's installed.
```
Comment (claudep) [inline django/db/backends/sqlite3/base.py]: Could that comment be removed, too?

**Comment by auvipy** [comment]
this is bad that I didn;t get any credit for my parts in this PR.

**Pull request merged**